lib/hammer_cli/output/adapter/abstract.rb

```diff
@@ -11,6 +11,10 @@ def initialize(context={}, formatters={})
       @formatters = HammerCLI::Output::Formatters::FormatterLibrary.new(filter_formatters(formatters))
     end
 
+    def paginate_by_default?
+      true
+    end
+
```

Comment:
Nit: I'd prefer to refactor to
```ruby 
def paginate_by_default?
    !!@paginate_by_default
end
```
and set the attribute in initializer, but it is minor thing